src/platform/web/ui/avatar.js

```diff
@@ -65,6 +70,8 @@ export class AvatarView extends BaseUpdateView {
         this._avatarLetterChanged();
         this._avatarTitleChanged();
         this._root = renderStaticAvatar(this.value, this._size);
+        const image = this._root.firstChild;
+        image?.addEventListener("error", () => this.setAvatarError());
```

Comment:
Perhaps also listen for load here and in both handlers, remove the listeners of both, so once an image is loaded or failed to load, there are no listeners anymore.